File: src/extension/compass/__tests__/fixtures/type_hints.py
class OrganizationService:
    pass


class Logger:
    pass


class Repository:
    pass


class TenantController:
    def __init__(self, service: OrganizationService):
        self.service = service

    def resolve(self, repo: Repository) -> OrganizationService:
        return self.service

    def count(self, value: int) -> str:
        return str(value)

File: src/extension/compass/__tests__/fixtures/type_hints.ts
export class OrganizationService {}

export class Logger {}

export class Repository {}

export class TenantController {
	private logger: Logger;

	constructor(private service: OrganizationService) {}

	resolve(repo: Repository): OrganizationService {
		return this.service;
	}

	load(): Array<Repository> {
		return [];
	}

	count(value: number): string {
		return String(value);
	}
}
